test/conditional-pino.test.js

```diff
@@ -0,0 +1,73 @@
+'use strict'
+
+const t = require('tap')
```

Comment:
Please use `node:test` instead. We have started converting to it, and it does not make sense to write new tests that will need to be converted later.

Reply:
@jsumners Will check this during this weekend. I believe node test runner does not yet support mocking modules.

Reply:
@karankraina that's incorrect: https://nodejs.org/api/test.html#mockmodulespecifier-options.

Reply:
@mcollina Thanks :) I'm learning something new everyday. Will update over this weekend.

Reply:
This is done. I took a slightly diff approach to test this.